client-ui/src/app/components/lineage/lineage-details/operation-properties-details/properties/properties.component.ts

```diff
@@ -49,6 +51,10 @@ export class PropertiesComponent implements OnDestroy {
     return getOperationColor(this.propertyType, this.propertyName)
   }
 
+  public propertiesContain(propertyType: PropertyType) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature: 'propertiesContain' to have a typedef](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4605551)